### ISSUE TITLE
Lots of dynamic notification banners when I invite a couple people on the doc 

### ISSUE THREAD

**Issue by virgile-dev** [numerique-gouv/impress#246]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**
If I invite a few people on the doc, one notification banner appears for each user. Which is annoying

**Describe the solution you'd like**
Can we make a single notification for the whole group or just change the UX. 
If invitations went through, just close the pane and notify me if there is an error.

**Comment by PanchoutNathan** [comment]
It has to come from the back first. Because it is impossible to add several members at once, we have to do a foreach. I think this is a pretty bad practice for several reasons. Like error handling or performance too. @sampaccoud @AntoLC

**Comment by AntoLC** [comment]
I think you can, when all the promises are made, we foreach them to show our toast popup, see:
https://github.com/numerique-gouv/impress/blob/1d1832b67a2453fefb298c9faad926f7c6865599/src/frontend/apps/impress/src/features/docs/members/members-add/components/AddMembers.tsx#L137-L148

I think we should keep this mechanism for `onError`, it is nice to know with which member the endpoint failed, but when it is successful, to show only 1 toast is probably enough, moreover they are listed in real time in your member list.

**Comment by PanchoutNathan** [comment]
Yes, it is clearly possible on the front side not to display several toasts. But I find it a shame to do so many "tricks" when it would be enough for the back to accept a batch of members to be added in a single POST. 

Because regarding performance or even simpler error management (because in the case where there are lots of errors, it amounts to displaying n toasts, or if we only display one error toast, we will not have the detail, except by tinkering)

But if you have to do the front version, no problem.

**Comment by virgile-dev** [comment]
With @rl-83 we worked on a revamp of the notifications. 
Could you detail here what we discussed and and link to the designs ?

**Comment by rl-83** [comment]
![Image](https://github.com/user-attachments/assets/b5aaac58-1615-43a8-b46a-927d4c904b30)

**Description**

This feature has been designed as part of the Subpages Update.

- Alerts appear at the bottom of the screen, in the center. They last for 5 seconds and then disappear permanently.
- A new alert takes the place of the old one (if another action is performed within 5 seconds). They do not stack.
- Alerts do not appear on mobile to avoid overloading the interface.
- Cases retained:
a. Confirmation of link copied to the clipboard
b. Confirmation of move to the trash, with the possibility of going back
c. Error uploading a video or file.
- The confirmation of title change for a document is removed.

[Reference prototype on Figma (FR)](https://www.figma.com/design/qdCWR4tTUr7vQSecEjCyqO/DOCS?node-id=7048-13564&t=tPhZGOWRvdZyfI9F-1)

**Comment by virgile-dev** [comment]
We no longer show those dynamic banners. Closing
